softdevice_controller/include/sdc.h

```diff
@@ -751,6 +728,50 @@ int32_t sdc_support_le_conn_cte_rsp_central(void);
  */
 int32_t sdc_support_le_conn_cte_rsp_peripheral(void);
 
+/** @brief Support for sending periodic advertising sync transfers as central role
+ *
+ * @note The application is required to call both @ref sdc_support_periodic_adv_sync_transfer_sender_central()
+ *       and @ref sdc_support_periodic_adv_sync_transfer_sender_peripheral() if both central and peripheral roles are supported.
+ *
+ * @retval 0               Success
+ * @retval -NRF_EPERM      This API must be called before @ref sdc_cfg_set() or @ref sdc_enable().
+ * @retval -NRF_EOPNOTSUPP sending periodic advertising sync transfers is not supported.
```

Comment:
```suggestion
 * @retval -NRF_EOPNOTSUPP Sending periodic advertising sync transfers is not supported.
```
Similar in other cases